### PR TITLE
Notify admins when a new resource of any type is created

### PR DESCRIPTION
**Pull request by punchagan**

### CODE REVIEW

File: backend/resources/gpml/config.edn [hunk 1]
```diff
@@ -223,16 +223,36 @@
                              :app-domain #duct/env "APP_DOMAIN"}}
   :gpml.handler.event/post-params {}
 
-  :gpml.handler.resource/post {:db #ig/ref :duct.database/sql}
+  :gpml.handler.resource/post {:db #ig/ref :duct.database/sql
+                               :mailjet-config
```
Comment (iperdomo) [inline backend/resources/gpml/config.edn:227]: it feels *strange* that I need to repeat the same configuration several times. Is not the case we want some key to hold that config and pass it around? 🤔 https://github.com/duct-framework/duct/wiki/Configuration#constants
e.g.

```
{[:duct/const :mailjet-config] {:api-key #duct/env "MAILJET_API_KEY"
                                :secret-key #duct/env "MAILJET_SECRET_KEY"
                                :app-name #duct/env "APP_NAME"
                                :app-domain #duct/env "APP_DOMAIN"}]
,,,
:gpml.handler.resource/post {:db #ig/ref :duct.database/sql
                             :mailjet-config #ig/ref :mailjet-config}
,,,
}
I have not tested it
```

**Review by iperdomo — CHANGES_REQUESTED** [review]
Let's try to find a way to not repeat the mailjet config several times, but define it once and use it via `#ig/ref`

File: backend/resources/gpml/config.edn (hunk 1 above)
Comment (punchagan) [reply backend/resources/gpml/config.edn:227]: Ah, yes, I was meaning to look into it, or ask you about it, but it skipped my mind. Thanks for the link, I'll look into this. :)

**Comment by iperdomo** [comment]
@punchagan I tested the code and is working fine using a *derived* key `{[:duct/const <your-key>] <value>}`

**Review by iperdomo — APPROVED** [review]

**Pull request merged**